server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -4784,7 +4790,7 @@ public void testSeqNoGenerator() throws IOException {
                     Versions.MATCH_ANY,
                     VersionType.INTERNAL,
                     Engine.Operation.Origin.PRIMARY,
-                    System.currentTimeMillis(), UNASSIGNED_SEQ_NO, 0);
+                    IndexRequest.UNSET_AUTO_GENERATED_TIMESTAMP, UNASSIGNED_SEQ_NO, 0);
```

Comment:
One more `startTime`, I think this should be:

```suggestion
                    System.nanoTime(), UNASSIGNED_SEQ_NO, 0);
```